soe/src/engineering.make

```diff
@@ -1,7 +1,7 @@
 core = 7.x
 api = 2
 
-projects[drupal][version] = "7.75"
+projects[drupal][version] = "7.78"
```

Comment:
I checked this over the weekend and it looks good. Need to re-start my Docksal stack to look at the search_api stuff.